source/manual/intro-to-govuk-docker.html.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Intro to Docker and govuk-docker
+section: Docker
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2019-07-01
+review_in: 6 months
+owner_slack: "#govuk-developers"
+---
+
+## Intro
+If you are new to Docker here's a quick intro in the context of the GOV.UK stack. There is also a wrapper for a lot of the docker commands in govuk-docker, which we will look at later on.
+
+
+### Install Docker
+
+*   Download Docker [https://www.docker.com/get-started](https://www.docker.com/get-started)
+*   Docker for Mac intro [https://docs.docker.com/docker-for-mac/](https://docs.docker.com/docker-for-mac/)
+
+### Video explaining the concepts in Docker
+
+[https://www.youtube.com/watch?v=i7yoXqlg48M](https://www.youtube.com/watch?v=i7yoXqlg48M) - first 30 minutes
+
+## Learning docker with content-publisher
+We'll run a few things here to get the [content-publisher] up and running in Docker. This is a convoluted example but it will help to explain the concepts involved.
+
+### Run a container from an image
+
+*   Make sure you have docker running, then run from the root of the [content-publisher] project:
```

Comment:
```suggestion
Make sure you have docker running, then run from the root of the content-publisher project:
```

Reply:
I've added a second step in this section. I know they're not all quite proper points, but I like to use the bullet points to indicate a step to make in the process, even if it's just one single step per headed section. Not sure if there's a better way to convey that.